RecoEgamma/EgammaIsolationAlgos/python/pfClusterIsolation_cfi.py

```diff
@@ -50,9 +54,14 @@
                                                       energyEndcap = cms.double(0)
                                                       )
 
+ootPhotonHcalPFClusterIsolationProducer = photonHcalPFClusterIsolationProducer.clone()
+ootPhotonHcalPFClusterIsolationProducer.candidateProducer = cms.InputTag('ootPhotons')
```

Comment:
Hi Slava,

The reducedEgammaProducer is run in the PAT step, so these edits will be added once I (re)open the AOD to miniAOD step.  See: https://github.com/kmcdermo/cmssw/commit/6d918f5ef9379f6226888be8f05fe5b6937dc23a